wiki/Client/Beatmap_editor/Menu/en.md

```diff
@@ -230,5 +230,14 @@
 
 | Name | Description |
 | :-- | :-- |
-| Show in-game help | Show the help overlay with key bindings and short explanations. Note: does not work anymore. |
+| Show in-game help | Show the help overlay with key bindings and short explanations.[^help-note] |
 | View FAQ... | Open the [Beatmapping](/wiki/Beatmapping) wiki article. |
+
+## Notes
+
+[^osz2-note]: This action is disabled for regular `osz` beatmaps.
+[^exit-note]: This action sometimes *ignores* changes made in the `Design` tab.
+[^stacking-note]: The stacking behaviour is controlled by the [stack leniency](/wiki/Beatmap/Stack_leniency) setting of the difficulty.
+[^live-mapping-note]: osu!, osu!taiko, and osu!catch use the osu!taiko key bindings.
```

Comment:
they don't use the respective mode keybinds??

Reply:
nope

Reply:
the reason is that the live mapping mode has been made for taiko specifically. since all three modes share the same editor, it _kind of_ works for all three, but only because of that